src/Drivers/Sms77.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Tzsk\Sms\Drivers;
+
+use Sms77\Api\Client;
+use Sms77\Api\Params\SmsParams;
+use Tzsk\Sms\Contracts\Driver;
+
+class Sms77 extends Driver
+{
+    protected array $settings;
+
+    protected Client $client;
+
+    public function __construct(array $settings)
+    {
+        $this->settings = $settings;
+        $this->client = new Client(
+            data_get($this->settings, 'apiKey'), 'Laravel-SMS-Gateway');
```

Comment:
What's that string? Wonder if someone would need to configure that.

Otherwise awesome job man. Thanks for contributing.

Reply:
Hi, it's just an identifier from which source the request is coming from. I took the name from `composer.json->description` and thought it would make sense to name it like this.

Reply:
Maybe adding it to the config Or if you don't think it needs to be configured, you can maybe add `app.url` or `app.name` config instead of the static string.

Reply:
Also if you could add a quick test that would be awesome.